quarto/src/Api/GameApi.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace App\Api;
+
+use App\Entity\Game;
+
+class GameApi {
```

Comment:
Why are you using a Class here? You can use functions directly (or static functions but it's worse)

Reply:
I don't know how to use fonctions whitout class with symfony
All trying to write a file wich is not a class lead to this error : 
![image](https://user-images.githubusercontent.com/39904906/41606012-037d5836-73e3-11e8-8f1d-9e623adec9f7.png)

Reply:
It's because you're importing "GameApi" class. As you use it, Symfony suppose that the corresponding class exists in file.

To avoid this error, you must import functions individually as described below:

use function mynamespace\trucfunction;

Or by using static functions.

More informations here: http://php.net/manual/fr/language.namespaces.importing.php